docs/api-reference.md

```diff
@@ -39,8 +88,13 @@ curl -XGET localhost:10101/index/user
 
 Creates an index with the given name.
 
+The request payload is in JSON, and may contain the `options` field. The `options` field is a JSON object with the following options:
+
+* `keys` (bool): Enables using column keys instead of column IDs.
+* `trackExistence` (bool): Enables or disables existence tracking on the index. Required for [Not](../query-language/#not) queries. It is `true` by default. Note that disabling track existence improves query performance.
```

Comment:
`Note that disabling track existence improves query performance.` That's not necessarily a true statement. Disabling `trackExistence` would improve write performance, but not necessarily query performance. I think if we want to include anything like this, it should be more along the lines of "Note that writing to an index with `trackExistence` enabled incurs a small performance penalty."